Lake/Util/MainM.lean

```diff
@@ -75,4 +75,8 @@ protected def error (msg : String) (rc : ExitCode := 1) : MainM α := do
 instance : MonadError MainM := ⟨MainM.error⟩
 instance [ToString ε] : MonadLift (EIO ε) MainM := ⟨MonadError.runEIO⟩
 instance : MonadLift IO MainM := inferInstance -- necessary, but don't know why
-instance : MonadLift LogIO MainM := ⟨fun x => liftM <| x.run MonadLog.eio⟩
+
+def _root_.Lake.LogIO.run (x : LogIO α) (verbosity := Verbosity.normal) : MainM α :=
```

Comment:
Since I am making suggestions, here is one nitpick: I would prefer this be named `runLogIO` and in this namespace (`MainM`).